dev/archery/archery/bot.py

```diff
@@ -187,46 +189,60 @@ def _clone_arrow_and_crossbow(dest, crossbow_repo, pull_request):
         Object containing information about the pull request the comment bot
         was triggered from.
     """
-    arrow_path = dest / 'arrow'
-    queue_path = dest / 'crossbow'
+    arrow_path = dest / "arrow"
+    queue_path = dest / "crossbow"
 
     # clone arrow and checkout the pull request's branch
-    pull_request_ref = 'pull/{}/head:{}'.format(
+    pull_request_ref = "pull/{}/head:{}".format(
         pull_request.number, pull_request.head.ref
     )
     git.clone(pull_request.base.repo.clone_url, str(arrow_path))
-    git.fetch('origin', pull_request_ref, git_dir=arrow_path)
+    git.fetch("origin", pull_request_ref, git_dir=arrow_path)
     git.checkout(pull_request.head.ref, git_dir=arrow_path)
 
     # clone crossbow repository
-    crossbow_url = 'https://github.com/{}'.format(crossbow_repo)
+    crossbow_url = f"https://github.com/{crossbow_repo}"
     git.clone(crossbow_url, str(queue_path))
 
     # initialize crossbow objects
-    github_token = os.environ['CROSSBOW_GITHUB_TOKEN']
+    github_token = os.environ["CROSSBOW_GITHUB_TOKEN"]
     arrow = Repo(arrow_path)
     queue = Queue(queue_path, github_token=github_token, require_https=True)
 
     return (arrow, queue)
 
 
 @crossbow.command()
-@click.argument('tasks', nargs=-1, required=False)
-@click.option('--group', '-g', 'groups', multiple=True,
-              help='Submit task groups as defined in tests.yml')
-@click.option('--param', '-p', 'params', multiple=True,
-              help='Additional task parameters for rendering the CI templates')
-@click.option('--arrow-version', '-v', default=None,
-              help='Set target version explicitly.')
+@click.argument("tasks", nargs=-1, required=False)
+@click.option(
+    "--group",
+    "-g",
+    "groups",
+    multiple=True,
+    help="Submit task groups as defined in tests.yml",
+)
+@click.option(
+    "--param",
+    "-p",
+    "params",
+    multiple=True,
+    help="Additional task parameters for rendering the CI templates",
+)
+@click.option(
+    "--arrow-version",
+    "-v",
+    default=None,
+    help="Set target version explicitly.",
+)
```

Comment:
You can see here that `black` adds a lot of unwanted vertical size...

Reply:
I don't like that either, but I can tolerate that in exchange of automatic formatting.

Reply:
Right. The rest of our Python codebase uses autopep8, so you may want to try it too.

Reply:
Running autopep8 has changed 3 lines in double aggressive mode. 
The output of `black` seems more homogeneous to me and it is quicker as well. 

I'd still prefer the more stricter black over autopep8, but I don't insist to it.